package.json

```diff
@@ -49,7 +49,7 @@
     "@changesets/changelog-github": "0.5.0",
     "@changesets/cli": "2.27.11",
     "@github/prettier-config": "^0.0.6",
-    "@types/node": "22.10.6",
+    "@types/node": "22.13.14",
```

Comment:
It looks like the latest version of node is 22.19.0, so rather than updating this manually could we instead just specify "22"?

Reply:
We don't use a major range value in package.jsons for security compliance reasons. @types/node doesn't mirror Node.js versions so we can't use 22.19.0 either. The latest version for @types/node@22 is 22.13.14, which I'm pinning to here.

Reply:
Ah cool, TIL that @types/node doesn't just point at the node version, thanks!